chains/btc/listener/util.go

```diff
@@ -0,0 +1,73 @@
+package listener
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/ChainSafe/sygma-relayer/e2e/evm"
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var RESOURCE_ID = evm.SliceTo32Bytes(common.LeftPadBytes([]byte{0}, 31))
```

Comment:
I would prefer this to be config based so each event handler per address has configurable resourceID.
So:
 - address A --> resource 0x...01
 - address B --> resource 0x....02
 
And do that in the event handler